src/app/public/modules/date-pipe/date.pipe.spec.ts

```diff
@@ -68,6 +68,15 @@ describe('Date pipe', () => {
     expect(expectedValues).toContain(value);
   });
 
+  it('should throw an error when provided an invalid date', () => {
+    try {
+      fixture.componentInstance.dateValue = 'foobar';
+      fixture.detectChanges();
```

Comment:
If this test _doesn't_ throw an error it would still pass. I'd either put a `fail()` call in the `try` block, or put the error-throwing method in the `expect` function.
```
expect(fixture.detectChanges).toThrow(new Error(...));
```

Reply:
Oops, you're right. I've updated this. Good catch.